test/e2e/inventory_v1beta2_http_test.go

```diff
@@ -386,4 +388,4 @@ func TestInventoryAPIHTTP_v1beta2_Host_WaitForSync(t *testing.T) {
 	}
 	_, err = client.KesselResourceService.DeleteResource(context.Background(), &delReq, opts...)
 	assert.NoError(t, err, "Failed to Delete Resource")
-}
+}*/
```

Comment:
It looks like our consistency e2e test for v1beta2 was caught in the crossfire of this block comment, we'll definitely want this enabled.

Reply:
Thank you, I will work on enabling it then. It is the cyclic dependency again that made me disable this test.

Reply:
Cool feel free to reach out if you have questions on changes